src/rustdoc-internals/rustdoc-test-suite.md

```diff
@@ -34,33 +32,41 @@ In this case, the start of the next line should be `//`, with no `@`.
 
 For example, `//@ !has 'foo/struct.Bar.html'` checks that crate `foo` does not have a page for a struct named `Bar` in the crate root.
 
+<!-- FIXME(fmease): Mention that the  regexes match case-sensitively and in single-line mode? -->
```

Comment:
Discussion: may also be worth mentioning if the regexes have any string and/or regex escape rules. For instance, I know some compiletest regex directives don't handle this very well.

Reply:
Hmm, I mean Python-flavored regex (which is mentioned) have all your typical Perl-style regex escape sequences, but that's to be expected. Regarding interaction with HtmlDocCk directive syntax, well, PATTERN is a shell-style quoted string (which is mentioned) whose escaping rules are pretty relaxed and thus there shouldn't really be a need to double-escape / worry about that.

I don't have the energy anymore to phrase this succinctly in-document and I think it can be deduced from "shell-style" + "Python-flavored".